src/terms/xc.jl

```diff
@@ -53,7 +55,31 @@ function ene_ops(term::TermXc, ψ, occ; ρ, kwargs...)
 end
 
 
-# Functionality for building the XC potential term and constructing the builder itself.
+function compute_kernel(term::TermXc; kwargs...)
+    error("Not implemented")
```

Comment:
just `Diagonal(kernel)` no? (for LDA at least)

Reply:
Yes for LDA it's just what I compute as "kernel" in the apply function. Not yet sure about GGA. I'll look at that tmr. I just wanted to get finite-differences to agree for the first checkpoint ;).

Reply:
GGA once you know what to do for applying the kernel you just have to convert G_to_r calls to G_to_r_matrix and so on. That would also be a nice testcase for forwarddiff. But again if it's too much trouble just forget about it...